tests/tck-build-logic/src/main/groovy/org.graalvm.internal.tck-harness.gradle

```
@@ -102,6 +103,59 @@ def matrixDefault = [
 
 final String METADATA_GROUP = "Metadata"
 
+List<String> getNewerVersionsFor(String library, String startingVersion) {
+    def baseUrl = "https://repo1.maven.org/maven2"
+    String group = library.split(":")[0].replace(".", "/")
+    String artifact = library.split(":")[1]
```

I'd extract the split operation:
```suggestion
    String[] libraryParts = library.split(":")
    String group = libraryParts[0].replace(".", "/")
    String artifact = libraryParts[1]
```